client/src/scene/ControlsUI.ts

```diff
@@ -1,28 +1,29 @@
 import { SceneGameArena } from "./SceneGameArena";
 
 export class ControlsUI {
-    constructor(scene: SceneGameArena, keys: Array<string>) {
-        let y = 600;
-        let index = 0;
-        const controlInfo = [
-            "Move Left",
-            "Move Right",
-            "Move Down",
-            "Rotate CCW",
-            "Rotate CW",
-        ];
+  constructor(scene: SceneGameArena, keys: Array<string>) {
+    let y = 550;
+    let index = 0;
+    const controlInfo = [
+      "Move Left",
+      "Move Right",
+      "Move Down",
+      "Rotate CCW",
+      "Rotate CW",
+      "Trade Action"
```

Comment:
Trade action is included into this file but the SVG is not being passed in? What is the purpose of keeping this here?